functions/venue.js

```diff
@@ -170,16 +170,16 @@ const createVenueData_v2 = (data, context) => ({
   name: data.name,
   config: {
     landingPageConfig: {
-      coverImageUrl: data.bannerImageUrl,
-      subtitle: data.subtitle,
-      description: data.description,
+      coverImageUrl: data.bannerImageUrl || "",
+      subtitle: data.subtitle || "",
+      description: data.description || "",
     },
   },
   theme: {
     primaryColor: data.primaryColor || DEFAULT_PRIMARY_COLOR,
   },
   host: {
-    icon: data.logoImageUrl,
+    icon: data.logoImageUrl || "",
```

Comment:
Not sure that setting these to a default empty string is an ideal choice. Wouldn't they be better/more explicitly/obviously `undefined`/checkable as such by leaving them as `undefined`.

Then at the point where we are actually inserting them into firestore, we can either replace those keys with the 'delete this key' firestore command, or enable the 'just ignore `undefined` and don't treat it as an error' option on this query itself (not globally).

@mike-lvov has used some of these patterns before, so if you need more specifics, may be able to ask him about it.

Reply:
I agree with you, I didn't know about the `ignoreundefinedproperies` setting. I'll use it, instead of flooding the DB with empty strings.

Reply:
@mike-lvov Do you have any experience with this? I tried adding `ignoreUndefinedProperies: true` in the config but I'm still getting rejected by the firebase functions, saying that undefined can not be used as a value.

Reply:
> I didn't know about the `ignoreundefinedproperies` setting

@denisdimitrov Yeah, I think we only found it relatively recently, possibly while you were on leave.

---

> but I'm still getting rejected by the firebase functions, saying that undefined can not be used as a value.

@denisdimitrov 😮😢 I'm not sure if we have actually used it yet. I think we ended up going with the delete pattern where I last called this out for a different use case.

Can you show code/errors/etc to help debug more?

Reply:
Nevermind, I'm an idiot. It's working now.

Reply:
Haha, what was the issue?

Reply:
I was changing the config of `admin.initializeApp` instead of `admin.firestore().settings` That's why it wasn't working.

Reply:
Cool, makes sense. A pitfall I will hopefully avoid myself in future now that I know! :p